src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -686,6 +686,7 @@ Microsoft.CodeAnalysis.Semantics.UnaryOperationKind.UnsignedPrefixIncrement = 77
 Microsoft.CodeAnalysis.SymbolDisplayFormat.RemoveGenericsOptions(Microsoft.CodeAnalysis.SymbolDisplayGenericsOptions options) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
 Microsoft.CodeAnalysis.SymbolDisplayFormat.RemoveLocalOptions(Microsoft.CodeAnalysis.SymbolDisplayLocalOptions options) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
 Microsoft.CodeAnalysis.SymbolDisplayFormat.RemoveMiscellaneousOptions(Microsoft.CodeAnalysis.SymbolDisplayMiscellaneousOptions options) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
+Microsoft.CodeAnalysis.SymbolDisplayFormat.WithTypeQualificationStyle(Microsoft.CodeAnalysis.SymbolDisplayTypeQualificationStyle typeQualificationStyle) -> Microsoft.CodeAnalysis.SymbolDisplayFormat
```

Comment:
Yes, this is a missing standard operation.

---
In reply to: [74125828](https://github.com/dotnet/roslyn/pull/13037#discussion_r74125828) [](ancestors = 74125828)